meetings/2024-04-10/minutes.md

```diff
@@ -4,36 +4,102 @@
+- [X] [Douglas Stebila (University of Waterloo)](https://github.com/dstebila)
 
 Other attendees
 
+ - [x] [Nigel Jones (IBM)](https://github.com/planetf1)
+ - [x] [Alex Bozarth (IBM)](https://github.com/ajbozarth)
+ - [x] [Michael (Max)imilien, IBM](https://github.com/maximilien)
+ - [x] [Ry Jones (Linux Foundation)](https://github.com/ryjones)
+ - [x] [Hart Montgomery, Linux Foundation](https://github.com/hartm)
+
+
 ## 1. Chair's introduction
+ - The chair thanks members for being patient as we get up and running
+ - TSC aims to be consensus driven & friendly
+ - TSC will discuss & make strategic decisions on technical matters. Developer call handlers regular operational matters
 
 ## 2. Approve agenda
 
+ - Agreed
+
 ## 3. Appoint minute-taker
 
+ - Nigel will take this week's notes. Agreed that we would ask at the beginning of each meeting & do in rotation.
+
 ## 4. Approval of minutes of last meeting
 
+- Last meeting's notes are open as [PR #14](https://github.com/open-quantum-safe/tsc/pull/14). Comments/reviewers invited. TSC members asked to approve. 
+- will use direct links to recordings in future minutes - portal not available to all (has been requested)
+
 ## 5. Voting procedure
 
+- [Issue #12](https://github.com/open-quantum-safe/tsc/issues/12) open for discussion
+- online/async voting to allow broadest engagement
+- open votes for technical matters
+- differing opinions on whether personal matters (including voting such as ranking) should be public or secret, or referred up to the board.
+- noted that most decisions consensus driven in any case
+- mechanism - git (simple), [gitvote](https://github.com/cncf/gitvote), [Helios](https://vote.heliosvoting.org/) are possible options
+- discussion to continue in [issue 12](https://github.com/open-quantum-safe/tsc/issues/12)
+
+
 ## 6. Addition of Vlad Gheorghiu to TSC
 
+- Vlad is maintainer of language wrappers 
+- Intent had been he was in the TSC from launch
+- Christan proposed, Basil seconded motion to add Vlad. Agreed by verbal vote of TSC members with no objections
+
 ## 7. Meeting cadence
 
+- Monthly still seems appropriate - lots of topics
+- Two suggestions:
+  - in PQCA TAC timeslot (alt. weeks) in off-week
+  - offline poll to accommodate most TSC members, and consider time rotation to handle differing timezones. This may require a higher frequency than monthly
+- Agreed poll will be conducted
+
 ## 8. Report from TAC representative (Thomas)
 
+- Project Lifecycle. 
+  - [Early draft](https://docs.google.com/document/d/1NV-0vNgXWdc81oqT0jv0C-9Funb8dySS06u90ghF-X4/edit)
+  - PQCA TAC suggests oqs tsc review and develop as most active project.
+  - objective is clarity for consumers as to what state the code is in - production, experimental, how good is security, maintenance - don't want to set wrong expectations. 
+  - Also covers adoption of new projects. Similar to apache incubation approach
+  - may apply at sub-project or component level
```

Comment:
Are you proposing to change the word "May" to "Will"?

Reply:
"must" or "needs to" seems to be more suitable than "will" given the original word was "may". Anyway, the core of the matter is: What designations should be applied to which projects or components is unclear to me.

Reply:
I think there's a few issues here
 - As minutes, these should reflect the discussion. If things are missed out, incorrect, don't adequately capture the discussion then absolutely we should update. Since there wasn't a firm decision I felt that using the word 'may' was appropriate in the minutes, but if I misrepresented, can you suggest alternate words?
 - The PQCA are delegating the creation of the lifecycle document to oqs as I understand it. We could take a copy of the initial draft, and collaboratively review/update through google docs etc, or via a github PR - whichever works best. Do you want to own it @baentsch ? @dstebila If no-one else does I can (but the reason it's come back to oqs is that you have the most experience and skin in the game..) , I think it's important. I think this is part of https://github.com/open-quantum-safe/tsc/issues/1 - ie it's more about policy & is fed back upstream to the pqca
  - https://github.com/open-quantum-safe/tsc/issues/2 is more a result of applying the policy - mapping to existing repos/capabilities